kratos/sources/variables.cpp

```diff
@@ -880,8 +763,6 @@ void KratosApplication::RegisterVariables() {
     KRATOS_REGISTER_VARIABLE(CONVECTION_COEFFICIENT)
 
     KRATOS_REGISTER_VARIABLE(SCALE)
-    KRATOS_REGISTER_VARIABLE(IS_JACK_LINK)
```

Comment:
I remove these variables because are registered already in the deprecated_variables.cpp. Without this is not possible to compile